index.js

```diff
@@ -74,7 +75,11 @@ var initConfig = function(projectConfig) {
   util._extend(config, require('./config.js').trace);
   util._extend(config, projectConfig);
   if (process.env.hasOwnProperty('GCLOUD_TRACE_LOGLEVEL')) {
-    config.logLevel = process.env.GCLOUD_TRACE_LOGLEVEL;
+    try {
+      config.logLevel = parseInt(process.env.GCLOUD_TRACE_LOGLEVEL);
+    } catch (e) {
+      // Just use the original log level if GCLOUD_TRACE_LOGLEVEL is malformed.
```

Comment:
Not familiar with internals but makes sense to defer when the logger isn't there.
I assume this is not breaking anything just failsafe. So just letting the user know (even a bit later) seems fine